src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -27,13 +27,19 @@ public static class DocumentDBSamples
         //   IAsyncCollector<T>
         //   ICollector<T>
         public static void InsertDocument(
-            [TimerTrigger("00:01")] TimerInfo timer,
-            [DocumentDB("ItemDb", "ItemCollection")] out ItemDoc newItem)
+            [TimerTrigger("00:01", RunOnStartup = true)] TimerInfo timer,
+            [DocumentDB("ItemDb", "ItemCollection")] out ItemDoc newItem,
```

Comment:
This was actually to verify my new caching code that I had to add. I wanted to make sure it would use the same underlying DocumentClient. I'll back all these changes out.

---
In reply to: [63893134](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63893134) [](ancestors = 63893134)